app/controllers/attachments_controller.rb

```diff
@@ -9,18 +11,31 @@ def new
 	end
 
 	def create
-		@attachment = Attachment.create( attachment_params )
-
-		if @attachment.save!
+		begin
+			@attachment = Attachment.new(attachment_params)
+			@attachment.save!
 			flash[:success] = "File uploaded!"
```

Comment:
@boyangles @duckgirl07  This is not rescuing the validation exception for some reason when you upload a non-PDF File.

@duckgirl07 When I click upload file with no file selected, it adds the docs/original/missing.png, which causes an error if you click on it